app/assets/stylesheets/application.scss

```diff
@@ -154,16 +154,20 @@
 @import 'templates/site';
 //templates
 
-//charts 
-@import 'charts/charts';
-//charts
-
 @import 'utilities/u-classes';
 
 @import 'custom';
 
 @import './map/custom-leaflet';
 
-@import './helpers/*';
-
-@import './components/*';
\ No newline at end of file
+@import './styles/base';
+@import './styles/helpers/**/*';
+@import './styles/components/icons';
```

Comment:
Is there a reason these are written out individually?

Reply:
The component styles are dependent on one another? Maybe this should be reflected in the folder structure... Or something should be moved to helper styles? I like the idea of the components being independent of each other. What do you think?

Reply:
Ok, I guess these would be resources in the nature map way of doing things. Will leave for now, but something to think about when the refresh is done.